src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationApiIntTest.kt

```diff
@@ -0,0 +1,247 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.prisonperson
+
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.integration.SqsIntegrationTestBase
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.persistence.repository.MigrationHistory
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.persistence.repository.MigrationHistoryRepository
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.MigrationStatus
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.MigrationType
+import java.time.LocalDateTime
+
+class PrisonPersonMigrationApiIntTest : SqsIntegrationTestBase() {
+
+  @Autowired
+  private lateinit var migrationHistoryRepository: MigrationHistoryRepository
+
+  @Nested
+  @DisplayName("/migrate/prisonperson/history")
+  inner class GetAllHistory {
```

Comment:
No actual code changes, I've just moved the tests for the UI endpoints to a separate test class to the migration tests as they won't change for each entity but the migration tests will.